src/Azure/Azure.Quantum.Client/SubmitterFactory.cs

```diff
@@ -27,11 +27,25 @@ public static class SubmitterFactory
                 "Microsoft.Quantum.Providers.Targets.MicrosoftSimulatorSubmitter, Microsoft.Quantum.Providers.Core",
                 "QirSubmitter"));
 
+        private static readonly ImmutableList<SubmitterInfo> QirPayloadGenerators = ImmutableList.Create(
+            new SubmitterInfo(
+                new Regex(@"\Amicrosoft\.simulator\.([\w]+\.)*[\w]+\z"),
+                "Microsoft.Quantum.Providers.Targets.MicrosoftSimulatorSubmitter, Microsoft.Quantum.Providers.Core",
+                "QirPayloadGenerator"));
```

Comment:
Just curious:
What is `\A` in the fragment `"\Amicrosoft`?

Reply:
It means that the match must occur at the start of the string. For example `microsoft.simulator.fullstate` would be a match while `Mmicrosoft.simulator.fullstate` would not.